### ISSUE TITLE
[BUG] BCryptGenRandom reference in function, error in building 

### ISSUE THREAD

**Issue by Snimm** [CCExtractor/ccextractor#1488]
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

To get the version of CCExtractor, you can use `--version`.

Please check all that apply and **remove the ones that do not**.

CCExtractor version: 0,94

# Necessary information

- Is this a regression (i.e. did it work before)? {YES/NO}
- What platform did you use? {Window}

# Additional information
I used the method given in the compilation guide for windows to build the project with visual studio. 
I got the following error:
```
1>ccx_rust.lib(std-c9a71bd02014cc03.std.7378dd35-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_BCryptGenRandom referenced in function _ZN3std6thread5local4fast12Key$LT$T$GT$14try_initialize17h880969def9aa54ebE
1>ccx_rust.lib(std-c9a71bd02014cc03.std.7378dd35-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_BCryptOpenAlgorithmProvider referenced in function _ZN3std3sys7windows4rand12fallback_rng17h3d523479d385cd68E
1>ccx_rust.lib(std-c9a71bd02014cc03.std.7378dd35-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_BCryptCloseAlgorithmProvider referenced in function _ZN3std3sys7windows4rand12fallback_rng17h3d523479d385cd68E
1>G:\ccex-github\ccextractor-master\ccextractor-master\windows\x64\Debug\ccextractorwin.exe : fatal error LNK1120: 3 unresolved externals
1>Done building project "ccextractor.vcxproj" -- FAILED.```

I know that the windows build is failing but this is a different error than the one I found in CI.

**Comment by PunitLodha** [comment]
This should've been fixed by #1480. Please check again, and let us know if the issue persists

**Comment by cfsmp3** [comment]
Closing since we think it's fixed. @Snimm feel free to reopen if needed.
